src/transformers/models/clap/processing_clap.py

```diff
@@ -39,13 +39,20 @@ class ClapProcessor(ProcessorMixin):
     def __init__(self, feature_extractor, tokenizer):
         super().__init__(feature_extractor, tokenizer)
 
-    def __call__(self, text=None, audios=None, return_tensors=None, **kwargs):
+    def __call__(
+        self,
+        text=None,
+        audios=None,
+        return_tensors=None,
+        **kwargs,
+    ):
```

Comment:
Should be able to fit on one line as we had before
```suggestion
    def __call__(self, text=None, audios=None, return_tensors=None, **kwargs):
```

Reply:
Done. Changed that by mistake earlier.